src/PlusDataCollection/CMakeLists.txt

```diff
@@ -1703,6 +1705,37 @@ ENDIF()
 TARGET_LINK_LIBRARIES(vtkPlusDataCollection ${PlusDataCollection_LIBS})
 
 # --------------------------------------------------------------------------
+# Optical Marker Tracking
+IF(PLUS_USE_OPTICAL_MARKER_TRACKING)
+  SET(OpticalMarkerTracking_SRCS
+    OpticalMarkerTracking/vtkPlusOpticalMarkerTracking.cxx
+    )
+
+  IF(MSVC OR ${CMAKE_GENERATOR} MATCHES "Xcode")
+    SET(OpticalMarkerTracking_HDRS
+      OpticalMarkerTracking/vtkPlusOpticalMarkerTracking.h
+      )
+  ENDIF()
+
+  SET(PlusDataCollection_INCLUDE_DIRS ${PlusDataCollection_INCLUDE_DIRS}
+    ${CMAKE_CURRENT_SOURCE_DIR}/OpticalMarkerTracking
+    ${aruco_INCLUDE_DIRS}
+    )
+
+  SET(PlusDataCollection_HDRS ${PlusDataCollection_HDRS}
+    ${OpticalMarkerTracking_HDRS}
+    )
+
+  SET(PlusDataCollection_SRCS ${PlusDataCollection_SRCS}
+    ${OpticalMarkerTracking_SRCS}
+    )
+
+  SET(PlusDataCollection_LIBS ${PlusDataCollection_LIBS}
+    OpticalMarkerTracking
```

Comment:
This doesn't seem right. Is there an external library for OpticalMarkerTracking? This seems like it should be aruco and OpenCV